src/south/SQLDbToFile/SQLDbToFile.class.spec.js

```diff
@@ -533,6 +533,15 @@ describe('SQLDbToFile', () => {
 
   it('should send uncompressed file when the result is not empty and compression is false', async () => {
     const RealDate = Date
+    global.Date = (...args) => {
+      switch (args.length) {
+        case 0:
+          return new RealDate(nowDateString)
+        default:
+          return new RealDate(...args)
```

Comment:
Why do we use a switch for one condition?

Reply:
Because of copy-paste :} I will rewrite it with if-else.